cms/static/js/views/active_video_upload_list.js

```diff
@@ -136,34 +138,46 @@ define([
                         uploadData.cid = model.cid; // eslint-disable-line no-param-reassign
                         uploadData.submit();
                     } else {
-                        $.ajax({
-                            url: this.postUrl,
-                            contentType: 'application/json',
-                            data: JSON.stringify({
-                                files: _.map(
-                                    uploadData.files,
-                                    function(file) {
-                                        return {file_name: file.name, content_type: file.type};
+                        _.each(
+                            uploadData.files,
+                            function(file, index) { // eslint-disable-line no-unused-vars
+                                $.ajax({
+                                    url: view.postUrl,
+                                    contentType: 'application/json',
+                                    data: JSON.stringify({
+                                        files: [{file_name: file.name, content_type: file.type}]
+                                    }),
+                                    dataType: 'json',
+                                    type: 'POST',
+                                    global: false   // Do not trigger global AJAX error handler
+                                }).done(function(responseData) {
+                                    _.each(
```

Comment:
Why we need a loop here? We are sending a separate ajax request for each file so i think we don't need loop here.